floss/main.py

```diff
@@ -125,6 +125,15 @@ def make_parser():
     parser.add_option("--save-workspace", dest="save_workspace",
                       help="save vivisect .viv workspace file in current directory", action="store_true")
 
+    shellcode_group = OptionGroup(parser, "Shellcode options", "Analyze BLOB containing shellcode")
```

Comment:
`BLOB` sounds like an acronym or something that the reader is supposed to know. maybe `raw binary file`?